src/app/playlists/page.tsx

```
@@ -86,6 +132,49 @@ export default function PlaylistsPage() {
+        {/* Create Playlist Modal */}
+        {isModalOpen && (
+          <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/80 backdrop-blur-sm p-4">
+            <div className="bg-zinc-900 border border-zinc-800 rounded-2xl w-full max-w-md p-6 shadow-xl">
+              <div className="flex justify-between items-center mb-6">
+                <h2 className="text-xl font-bold">新規プレイリスト作成</h2>
+                <button
+                  onClick={() => setIsModalOpen(false)}
+                  className="text-zinc-400 hover:text-white transition-colors"
+                >
+                  <X className="h-6 w-6" />
+                </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)